removeFromArray/removeFromArray.js

```diff
@@ -2,4 +2,31 @@ const removeFromArray = function() {
 
 }
 
+/* Try without using forEach()
+
+const removeFromArray = function(array, ...toRemove) {
```

Comment:
I had to look up the `...` preceding a function argument name. You're teaching _me_ stuff!